pkg/azure/instrument/instrument.go

```diff
@@ -50,6 +52,7 @@ func RecordAzAPIMetric(err error, azServiceName string, invocationTime time.Time
 
 	// compute the time taken to complete the AZ service call and record it as a metric
 	elapsed := time.Since(invocationTime)
+	fmt.Printf("Elapsed time: %v\n", elapsed)
```

Comment:
```suggestion
```
We can remove this imo

Reply:
ah this was left, removed and pushed the changes.